### PR TITLE
Rewrite Dashboard introduction page

### PR DESCRIPTION
**Pull request by debrisapron**
Addresses #4661. Attempts to punch-up the content with more images, more links, more emphasis on value-add, more lead-in to the various features & a little more technical detail for a dev audience. The idea is to hit a midpoint between just a list of features and an in-depth technical discussion - just enough information to create intrigue.

### CODE REVIEW

File: content/guides/dashboard/introduction.md [hunk 1]
```diff
@@ -2,10 +2,11 @@
 title: Dashboard
 ---
 
-The [Cypress Dashboard](https://on.cypress.io/dashboard) is a service that gives
-you access to recorded test results - typically when running Cypress tests from
-your [CI provider](/guides/continuous-integration/introduction). The Dashboard
-provides you insight into what happened when your tests ran.
+The [Cypress Dashboard Service](https://on.cypress.io/dashboard) is our
+Enterprise-Ready web-based companion to the Cypress app. It gives you online
```
Comment (pstakoun) [inline content/guides/dashboard/introduction.md:6]:
```suggestion
enterprise-ready, web-based companion to the Cypress app. It gives you online
```

File: content/guides/dashboard/introduction.md [hunk 2]
```diff
@@ -2,10 +2,11 @@
 title: Dashboard
 ---
 
-The [Cypress Dashboard](https://on.cypress.io/dashboard) is a service that gives
-you access to recorded test results - typically when running Cypress tests from
-your [CI provider](/guides/continuous-integration/introduction). The Dashboard
-provides you insight into what happened when your tests ran.
+The [Cypress Dashboard Service](https://on.cypress.io/dashboard) is our
```
Comment (pstakoun) [inline content/guides/dashboard/introduction.md:5]:
```suggestion
The [Cypress Dashboard](https://on.cypress.io/dashboard) is our
```

File: content/guides/dashboard/introduction.md [hunk 3]
```diff
@@ -2,10 +2,11 @@
...
+Enterprise-Ready web-based companion to the Cypress app. It gives you online
+access to your recorded test results, orchestrates test runs across multiple
+machines, provides rich analytics and diagnostics, and integrates those insights
+with your favorite SaaS products.
```
Comment (pstakoun) [inline content/guides/dashboard/introduction.md:9]:
```suggestion
with your favorite tools.
```

File: content/guides/dashboard/introduction.md [hunk 4]
```diff
@@ -29,80 +30,83 @@ Check out the <Icon name="github"></Icon>
...
-- Get the entire stack trace of failed tests.
-- View screenshots taken when tests fail or when using
-  [`cy.screenshot()`](/api/commands/screenshot).
-- Watch a video of your entire test run or a video clip at the point of test
-  failure.
-- See how fast your spec files ran within CI including whether they were run in
-  parallel.
-- See related groupings of tests.
+Store the full history of your test results, with
+[video clips, screenshots and full stack traces](/guides/dashboard/runs#Test-failures).
```
Comment (pstakoun) [inline content/guides/dashboard/introduction.md:38]:
```suggestion
[video clips, screenshots, and full stack traces](/guides/dashboard/runs#Test-failures).
```

File: content/guides/dashboard/introduction.md [hunk 5]
```diff
@@ -29,80 +30,83 @@ Check out the <Icon name="github"></Icon>
...
+Quickly see the current state of your app on the
+[Latest Runs](/guides/dashboard/runs) page, identify problematic trends with
+rich [Analytics](/guides/dashboard/analytics), and diagnose unreliable tests
+with our [Flaky Test Management](/guides/dashboard/flaky-test-management)
+feature. Associate related tests with
```
Comment (pstakoun) [inline content/guides/dashboard/introduction.md:43]:
```suggestion
with [Flaky Test Management](/guides/dashboard/flaky-test-management).
Associate related tests with
```

File: content/guides/dashboard/introduction.md [hunk 6]
```diff
@@ -29,80 +30,83 @@ Check out the <Icon name="github"></Icon>
...
-From the Dashboard you can:
-
-- Integrate Cypress into Slack on every recorded test run.
-
-#### See test runs in the Cypress Project Runs
-
-Additionally we've integrated the latest test run info into
-[Cypress](/guides/core-concepts/cypress-app). This means you can see the status
-of your tests in the _Runs_ tab from within each project.
+For users of the [Cypress App](/guides/core-concepts/cypress-app), we've
```
Comment (pstakoun) [inline content/guides/dashboard/introduction.md:49]:
```suggestion
For users of the [Cypress app](/guides/core-concepts/cypress-app), we've
```

File: content/guides/dashboard/introduction.md [hunk 7]
```diff
@@ -29,80 +30,83 @@ Check out the <Icon name="github"></Icon>
...
+[Business or Enterprise plans](https://www.cypress.io/pricing/) can integrate
+securely with
+[Self-managed GitLab](https://docs.gitlab.com/ee/subscriptions/self_managed/)
+instances too.
+
+<DocsImage src="/img/dashboard/github-integration/status-checks-per-spec.png" alt="Status checks per spec"></DocsImage>
+
+### Collaborate and organize
+
+We also integrate with two of the world's most popular collaboration tools,
```
Comment (pstakoun) [inline content/guides/dashboard/introduction.md:92]:
```suggestion
We also integrate with two of the world's most popular collaboration tools:
```

File: content/guides/dashboard/introduction.md (hunk 2 above)
Comment (mattvCypress) [reply content/guides/dashboard/introduction.md:5]: 👍

File: content/guides/dashboard/introduction.md (hunk 1 above)
Comment (mattvCypress) [reply content/guides/dashboard/introduction.md:6]: 👍

File: content/guides/dashboard/introduction.md (hunk 3 above)
Comment (mattvCypress) [reply content/guides/dashboard/introduction.md:9]: 👍

File: content/guides/dashboard/introduction.md (hunk 4 above)
Comment (mattvCypress) [reply content/guides/dashboard/introduction.md:38]: 👍

File: content/guides/dashboard/introduction.md (hunk 5 above)
Comment (mattvCypress) [reply content/guides/dashboard/introduction.md:43]: 👍

File: content/guides/dashboard/introduction.md (hunk 6 above)
Comment (mattvCypress) [reply content/guides/dashboard/introduction.md:49]: 👍

File: content/guides/dashboard/introduction.md (hunk 7 above)
Comment (mattvCypress) [reply content/guides/dashboard/introduction.md:92]: 👍

**Review by mattvCypress — APPROVED** [review]

File: content/guides/dashboard/introduction.md (hunk 4 above)
Comment (debrisapron) [reply content/guides/dashboard/introduction.md:38]: https://www.youtube.com/watch?v=P_i1xk07o4g

File: content/guides/dashboard/introduction.md [hunk 8]
```diff
@@ -29,80 +30,83 @@ Check out the <Icon name="github"></Icon>
...
+[video clips, screenshots, and full stack traces](/guides/dashboard/runs#Test-failures).
+Quickly see the current state of your app on the
+[Latest Runs](/guides/dashboard/runs) page, identify problematic trends with
+rich [Analytics](/guides/dashboard/analytics), and diagnose unreliable tests
+with [Flaky Test Management](/guides/dashboard/flaky-test-management). Associate
+related tests with
+[grouping](/guides/guides/parallelization#Grouping-by-browser) to see results
+broken down by browser and OS.
 
 <DocsImage src="/img/dashboard/dashboard-runs-list.png" alt="Dashboard Screenshot" ></DocsImage>
```
Comment (pstakoun) [inline content/guides/dashboard/introduction.md:47]: We should update this screenshot

**Review by pstakoun — APPROVED** [review]

**Pull request merged**